### ISSUE TITLE
Multi-select devices on same device style view

### ISSUE THREAD

**Issue by deku-m** [peterbaumert/netbox-device-view#26]
Some devices have the same overview style as others, is it maybe option to have like a multi-select?
 
![afbeelding](https://github.com/peterbaumert/netbox-device-view/assets/37069737/c00081a7-24f7-46c1-b590-a1c1c3459729)

As now you need to create seperate device types with sometimes the same style.

**Comment by peterbaumert** [comment]
You mean like having patchpanels from two different manufacturers, but port naming and so on is same so you can say this css is for both the same?

**Comment by deku-m** [comment]
Yes, for example the firepower 1120/1140 are identical with ports, as also a 9200/L 9300/L with 24/48ports layout if im not mistaken
